tests/framework/test_values.py

```diff
@@ -2,14 +2,21 @@
 import pandas as pd
 import pytest
 
-from vivarium.framework.values import ValuesManager, list_combiner, union_post_processor
+from vivarium.framework.values import (
+    ValuesManager,
+    list_combiner,
+    rescale_post_processor,
+    union_post_processor,
+)
 
 
 @pytest.fixture
 def manager(mocker):
     manager = ValuesManager()
     builder = mocker.MagicMock()
-    builder.step_size = lambda: lambda: pd.Timedelta(days=1)
+    builder.time.simulant_step_sizes = lambda: lambda idx: pd.Series(
+        pd.Timedelta(days=3), index=idx
+    )
```

Comment:
It'd be a better test if the time-deltas were not all the same

Reply:
how is this?

Reply:
Just because this means that we can test that it is in fact acting on each simulant individually.

Reply:
Oh, I agree, I meant how did the addition I made (to use 3 days for even and 5 days for odd) look, to be responsive to your comment